### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by surbhi84**

### CODE REVIEW

**Review by bharatjain7723 — APPROVED** [review]
Looks good

File: components/navigation/index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+              src="./icons/heartFilled.svg"
+              alt="heart icon"
+              class="nav-icon"
+            />
+            Wishlist
+          </li>
+          <li class="nav-link">
+            <a href=""></a>
+            <img src="./icons/cart.svg" alt="cart icon" class="nav-icon" /> Cart
+          </li>
```
Comment (hiteshverma27) [inline components/navigation/index.html]: Put elements like logo and nav-links in `<a>` tag. rest is good

**Review by hiteshverma27 — APPROVED** [review]

File: components/navigation/index.html (hunk 1 above)
Comment (surbhi84) [reply components/navigation/index.html]: yeah thanks..done now!

**Comment by surbhi84** [comment]
got 2 approves..will merge now.

**Pull request merged**